### ISSUE TITLE
Details about the methods - merging the cohort-level objects into pan-cancer objects

### ISSUE THREAD

**Issue by zvittorio** [ding-lab/PanCan_snATAC_publication#4]
Dear authors, 

Congratulation for this huge study on cancer transitions, which I loved reading. 
In the Methods section, paragraph 'Merging of sc/snRNA-seq data across samples', you write that cancer-cohort-level objects were merged to generate pan-cancer objects. In particular

> To make a merged object with all cell types, 600 cells were randomly sampled for each cohort-level cluster, and the resulting set of cells was used for merging. The resulting normalized matrices of merged objects were used for subsequent analysis.

I didn't fully understand the random selection of 600 cells from cohort-level clusters. What was the reason for that, and what do you mean by "used for merging"?
From what I understood you select 600 cells at random from each cluster to create a big pan-cancer object with all cell types. But then you use all the cells for showing the UMAPs in the Extended Figures (1 and 2)?
If that is the case, does it mean that you used that lower number of cells for all the other analysis involving sc/snRNA-seq data?

Thank you for clarifying!

Vittorio

**Comment by nvterekhanova** [comment]
Hi Vittorio,

For this study we created several merged RNA objects: (1) cohort level objects, and two pan-cancer objects: (2) with cancer and selected normal cells, potential cells-of-origin, and (3) with all cell types. For pan-cancer RNA object (3) with all cell types we had to down sample cells due to limitations of R and to reduce computational complexity. In the Extended Figs 1-2 we show UMAPs for objects (1) and (2). For all DEGs analysis we used merged pan-cancer object (2).


Nadezhda
